### DISCUSSION TITLE
Server Version 2024.2.3

### DISCUSSION THREAD

**Discussion by martinjgrunwald** [dani-garcia/vaultwarden#4443] [General]
The official BitWarden Server is already at Version 2024.2.3 while Vaultwarden is still at 2024.1.2. I don't really care about new features but currently I have occasional issues with the Browser extension. Looking at [this issue](https://github.com/bitwarden/clients/issues/5407) this seems to be down to the different server version used here. 
I read that there will be breaking changes in the upcoming v1.31.0. Will this version update the server to 2024.2.3 and do you have a timeline for the release?

**Comment by BlackDex** [comment]
First, no, there are no timelines, milestones, eta's etc...

Second. The is PR already in the [bw_web_builds](https://github.com/dani-garcia/bw_web_builds/) which needs to be checked a bit more careful.

**Comment by martinjgrunwald** [comment]
Thank you for your reply! Take your time for the quality control. Somehow switching to MS Edge doesn't cause this problem even though it is the same browser under the hood 🤨 so I have a workaround until the new web vault hopefully fixes the issue in Chrome 👍

**Reply by BlackDex** [reply]
The browser extension btw has nothing to do with the web-vault.
None of the other clients btw have anything to do with the web-vault. These do not use that at all.
You can even disable the web-vault and all other clients would still work.

**Reply by martinjgrunwald** [reply]
Do you have an idea what could cause the weird issue within Chrome instead?

**Reply by BlackDex** [reply]
Nope not really. Those are client side items as far as i can tell. And that is not something this project maintain.
Also, that bug reports is talking about the hosted Bitwarden service as far as i can tell. Nothing to do with Vaultwarden.

You must be more specific on what issue you have and how it appears.
